Venmo/src/test/java/com/braintreepayments/api/VenmoActivityResultContractUnitTest.java

```diff
@@ -61,6 +62,17 @@ public void createIntent_returnsIntentWithExtras() throws JSONException {
         JSONAssert.assertEquals(expectedBraintreeData, new JSONObject(intent.getStringExtra(EXTRA_BRAINTREE_DATA)), JSONCompareMode.STRICT);
     }
 
+    @Test
+    public void createIntentForFallbackToWebWithVenmoInstalled_returnsIntentWithExtras() throws JSONException {
```

Comment:
The title of this test doesn't really match the assertion - should it be null or should we assert on the extras?

Reply:
I think this can be removed entirely, we had originally built the URL and appended it to the intent but don't have that logic in the VenmoAcitivityResultContract at all anymore. Removing this and will also update the missing test cases.